Library/Contracts.ts

```diff
@@ -10,6 +10,7 @@ export module Contracts {
         SQL = 0,
         Http = 1,
         Other = 2,
+        ApplicationInsights = 3,
```

Comment:
To add to what I mentioned above. This class can be removed completely. We shouldn't depend on this anymore.